### PR TITLE
githubteacher adds Orlando hometown file

### PR DESCRIPTION
**Pull request by githubteacher**
This PR will add content for the Orlando, FL hometown file ☀️ 

Some open ❓ before I merge:

1. if you haven't been to Orlando, what are you looking forward to?
1. if you have been to Orlando, what did you enjoy? what did dislike?
1. if you have small children, what sorts of activities do you enjoy doing with them?
1. @brianamarie when you visit other cities, do you like going to theaters or seeing 🎼  performances?

### CODE REVIEW

**Comment by brianamarie** [comment]
@githubteacher I'd love to see some live music in Orlando!

File: githubteacher-orlando.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Orlando, FL
+
+### Restaurants
+- The Pharmacy
```
Comment (mmeans01) [inline githubteacher-orlando.md]: Is that a place to eat?  What's it like?

**Comment by msowrierajan** [comment]
Enjoyed disney world when visited last time

File: githubteacher-orlando.md (hunk 1 above)
Comment (githubteacher) [inline githubteacher-orlando.md]: @mmeans01 the Pharmacy is a super secret hideout, it **is** a restaurant, I'll add more info to my file.

**Comment by githubteacher** [comment]
@brianamarie @msowrierajan @mmeans01 I addressed your feedback in commit id 991d22e

**Comment by mmeans01** [comment]
:+1:

**Comment by vivekmanand** [comment]
👍

**Pull request merged**